string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,9 @@
+/* exported isVowel */
+function isVowel(char) {
+  const vowels = [
+    'a', 'e', 'i', 'o', 'u',
+    'A', 'E', 'I', 'O', 'U'
```

Comment:
To avoid adding lower and upper case to the `vowels` array you could use `toLowerCase` on `char` then do the comparison.